joomla/joomla-cms
# [4.0] Fix email to friend popup layout

Pull Request for Issue #18612 .

### Summary of Changes
Fixes layout to component popups (eg. email to friend).


### Testing Instructions
Apply PR and navigate to 'Single Article' in frontend and check 'Email to Friend' (Cog -> Email)


### Expected result



### Actual result



### Documentation Changes Required
